winchan.js

```diff
@@ -197,7 +197,9 @@ var WinChan = (function() {
                 cb = null;
               }
             }
-          } catch(err) { }
+          } catch(err) {
+            console.err(err);
```

Comment:
we were loosing all the errores there (even in the callback) that is why I changed it but sounds good, will change that